src/index.html

```diff
@@ -5,8 +5,382 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500;700;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header" id="home">
+      <a href="#home" class="header__img">
+        <img src="./images/logo.png" alt="picture">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link nav__link--is-active is-active"
+                href="#"
+              >
```

Comment:
fix code style everywhere 

```suggestion
            <a
              class="nav__link nav__link--is-active is-active"
              href="#"
            >
```